indexprovider/wrapper.go

```diff
@@ -262,6 +263,14 @@ func (w *Wrapper) IndexerAnnounceAllDeals(ctx context.Context) error {
 	return merr
 }
 
+func (w *Wrapper) IndexerAnnounceLatest(ctx context.Context) (cid.Cid, error) {
+	e, ok := w.prov.(*engine.Engine)
```

Comment:
Double checking `w.prov` will never be `nil`?

Reply:
It will never be nil 👍